testing/src/main/java/org/oppia/android/testing/story/StoryProgressTestHelper.kt

```diff
@@ -346,14 +403,67 @@ class StoryProgressTestHelper @Inject constructor(
   }
 
   /**
-   * Marks the only chapter of test topic 1 story 2 as recently played. For specifics on parameters
-   * and nuances, see: [markRecentlyPlayedTestTopic0Story0Exp0].
+   * Marks the second chapter of test topic 0 story 0 as in progress not saved. For specifics on
+   * parameters and nuances, see: [markInProgressNotSavedTestTopic0Story0Exp0].
    */
-  fun markRecentlyPlayedTestTopic1Story2Exp0(
+  fun markInProgressNotSavedTestTopic0Story0Exp1(
```

Comment:
Add description of arguments in kdoc using  `@param `

Reply:
done